assets/components/LogContainer.vue

```diff
@@ -73,6 +81,11 @@ export default {
       return config.base;
     },
   },
+  methods: {
```

Comment:
I recommend putting the keyboard shortcut here to be closer to the button.

Reply:
Done.